packages/common-all/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
+    // "noUncheckedIndexedAccess": true,
```

Comment:
remove if not used

Reply:
[`b73bd72` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/b73bd72951eca8b7183630febd438f54adec46a5)